test-helper/index.js

```diff
@@ -1,18 +1,26 @@
-function context(description, spec) {
+const $ = require('jquery');
```

Comment:
Could we try to drop the jQuery? Or it would be in another PR?

Reply:
It would be in another PR